tests/scripts/analyze_outcomes.py

```diff
@@ -214,6 +214,324 @@ def do_analyze_driver_vs_reference(results: Results, outcome_file, args):
             }
         }
     },
+    'analyze_driver_vs_reference_cipher_aead': {
+        'test_function': do_analyze_driver_vs_reference,
+        'args': {
+            'component_ref': 'test_psa_crypto_config_reference_cipher_aead',
+            'component_driver': 'test_psa_crypto_config_accel_cipher_aead',
+            # Ignore suites that are being accelerated
+            'ignored_suites': [
+                'aes.cbc',
+                'aes.cfb',
+                'aes.ecb',
+                'aes.ofb',
+                'aes.rest',
+                'aes.xts',
+                'aria',
+                'camellia',
+                'ccm',
+                'chacha20',
+                'chachapoly',
+                'cipher.aes',
+                'cipher.aria',
+                'cipher.camellia',
+                'cipher.ccm',
+                'cipher.chacha20',
+                'cipher.chachapoly',
+                'cipher.des',
+                'cipher.gcm',
+                'cipher.nist_kw',
+                'cipher.padding',
+                'des',
+                'gcm.aes128_de',
+                'gcm.aes128_en',
+                'gcm.aes192_de',
+                'gcm.aes192_en',
+                'gcm.aes256_de',
+                'gcm.aes256_en',
+                'gcm.camellia',
+                'gcm.misc',
+                'cmac',
+            ],
+            'ignored_tests': {
+                # Following tests depends on AES_C/DES_C
+                'test_suite_pem': [
+                    'PEM read (AES-128-CBC + invalid iv)'
```

Comment:
Not a blocker, but we'll need a follow-up cleanup when both #8398 and #8303 are merged.

Reply:
Yes, my intention was to update 8398 if this one gets merged first. I even had a local commit doing that: https://github.com/Mbed-TLS/mbedtls/commit/4207c655e02add507a5510a88261dbc5ea7293b0 - just waiting for this one to get merged before applying it. I'm labeling 8398 as "needs-preceding-pr" as a reminder.